benchmark/src/main/java/com/datatorrent/benchmark/window/README.md

```diff
@@ -0,0 +1,29 @@
+Windowed Operator Benchmark Readme
+
+Introduction
+The Windowed operator benchmark are applications to test the performance of windowed operator. 
+This benchmark applications only available for Malhar 3.7.0 or later versions. 
+The source code is located inside the malhar/benchmark. There are two applications, 
+one is benchmark for WindowedOperator, another is for KeyedWindowedOperator. 
+
+Steps of compile
+- Get source code of malhar
+- Compile the malhar with all module option "-Pall-modules": "mvn clean install -Pall-modules -DskipTests"
+
+Run the benchmark and check the result in cluster
+- Start apex cli
```

Comment:
Is there a need to perform cleanup of existing state before running the benchmark?